src/back2top.less

```diff
@@ -0,0 +1,20 @@
+.to-top {
+    opacity: 1;
+    display: inline-block;
+    padding: 1em;
+    position: fixed;
+    bottom: 1em;
+    right: 1em;
+    .paper-btn {
+    padding: 0.6em 1em;
+    background: white;
+    border-top-left-radius: 185px 160px;
```

Comment:
As @TotomInc  was saying i think this should use the .border class so it makes the same style across the framework

Reply:
I didn't use .border in order to make the button a circle. Adding .border would make it like a square (which is consistent across framework)
So should I
1. Make the button square - using `.border' class?
2. Define 'circle-border' in borders.less and keep using circle button?